### ISSUE TITLE
There can be only one type named "Mutation" invalid-graphql-schema

### ISSUE THREAD

**Issue by guiaramos** [cjoudrey/graphql-schema-linter#218]
Is it possible to allow multiple Mutation?

**Comment by cjoudrey** [comment]
Hi @guiaramos, I'm not sure I fully understand what you're asking. Can you share a snippet that is causing this validation error?

**Comment by cjoudrey** [comment]
Feel free to re-open this if it is still an issue.

**Comment by guiaramos** [comment]
Sorry for my bad explanation @cjoudrey, I am facing this issue:

```sh
/Users/guilhermeramos/Desktop/projects.nosync/snuper/server/src/schemas/Profile/profile.graphql
1:6 There can be only one type named "Query".     invalid-graphql-schema
1:6 There can be only one type named "Query".     invalid-graphql-schema
6:6 There can be only one type named "Mutation".  invalid-graphql-schema
6:6 There can be only one type named "Mutation".  invalid-graphql-schema
```

but actually it's not a error as we have multiple mutation and query types and then we use mergeType and mergeResolvers from https://github.com/Urigo/merge-graphql-schemas

Is it possible to ignore this error of "There can be only one type Mutation/Query"?

Ty!

**Comment by cjoudrey** [comment]
Thanks for the clarification @guiaramos! 😄 

Here's a similar issue about `merge-graphql-schemas`: https://github.com/cjoudrey/graphql-schema-linter/issues/147#issuecomment-418152380

Let's continue the discussion there.

Thanks! 😄
